scripts/compare_deb_repo.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+import requests
+from debian.deb822 import Packages
+from debian.changelog import Changelog
+
+DISTS = [os.path.basename(d) for d in glob.glob('debian/*') if d != 'debian/buildscripts']
```

Comment:
I like using `pathlib.Path`
```suggestion
DISTS = [p.name for p in Path('debian').glob('*') if p.name != 'buildscripts']
```